test/CMakeLists.txt

```diff
@@ -96,17 +96,27 @@ function(set_pit_tests_properties)
        PIR_OP_TESTS)
   foreach(IR_OP_TEST ${PIR_OP_TESTS})
     if(TEST ${IR_OP_TEST})
-      set_tests_properties(
```

Comment:
This will lead to property overwritten by later settings and cause environment variables invaid.